spec.html

```diff
@@ -181,19 +181,29 @@ <h1>RealmImportValue ( _specifierString_, _exportNameString_, _callerRealm_, _ev
 				1. Assert: Type(_exportNameString_) is String.
 				1. Assert: _callerRealm_ is a Realm Record.
 				1. Assert: _evalRealm_ is a Realm Record.
-				1. Let _promiseCapability_ be ! NewPromiseCapability(%Promise%).
+				1. Let _innerCapability_ be ! NewPromiseCapability(%Promise%).
 				1. Let _runningContext_ be the running execution context.
 				1. If _runningContext_ is not already suspended, suspend _runningContext_.
 				1. Push _evalContext_ onto the execution context stack; _evalContext_ is now the running execution context.
-				1. Perform ! HostImportModuleBindingDynamically(_callerRealm_, _evalRealm_, _specifierString_, _exportNameString_, _promiseCapability_).
+				1. Perform ! HostImportModuleDynamically(*null*, _specifierString_, _innerCapability_).
```

Comment:
I really like this `HostImportModuleDynamically` goes well with the running execution context shifting to the target realm and restored after.